content/docs/guides/observability/metrics.md

```diff
@@ -197,13 +203,176 @@ Both metrics have the following labels:
 
 In addition, the `osm_request_total` metric has a `response_code` label representing the HTTP status code of each request, e.g. `200`, `404`, etc.
 
-##### Known Gaps
+#### Known Gaps
 
 - HTTP requests that invoke a local response from Envoy have "unknown" `destination_*` labels on metrics.
   - In the demo, this includes requests from the bookthief to the bookstore.
 - Metrics are only recorded for traffic where both endpoints are part of the mesh. Ingress and egress traffic do not have statistics recorded.
 - Metrics are recorded in Prometheus with all instances of '-' and '.' in tags converted to '\_'. This is because proxy-wasm adds tags to metrics through the name of the metric and Prometheus does not allow '-' or '.' in metric names, so Envoy converts them all to '\_' for the Prometheus format. This means a pod named 'abc-123' is labeled in Prometheus as 'abc_123' and metrics for pods 'abc-123' and 'abc.123' would be tracked as a single pod 'abc_123' and only distinguishable by the 'instance' label containing the pod's IP address.
 
+### Control Plane
+
+The following metrics are exposed in the Prometheus format by the OSM control plane components. The `osm-controller` and `osm-injector` pods have the following Prometheus annotation.
+
+```yaml
+annotations:
+   prometheus.io/scrape: 'true'
+   prometheus.io/port: '9091'
+```
+
+<table>
```

Comment:
For the sake of whoever has to update this later, does a markdown table work here instead?